project-base/templates/Front/Inline/Newsletter/subscription.html.twig

```diff
@@ -29,4 +29,4 @@
         </div>
     </div>
 {{ form_end(form) }}
-{{ init_js_validation(form, true) }}
+{#{ init_js_validation(form, true) }#}
```

Comment:
please check if this should be here and remove the comment or remove the whole line

Reply:
Fixed